stories/toolbar/toolbar.stories.js

```diff
@@ -99,7 +99,7 @@ export const overflow = () => `<div style="height:250px">
         <button class="fd-button fd-button--compact fd-button--transparent">Save</button>
         <button class="fd-button fd-button--compact fd-button--transparent">Copy</button>
         <span class="fd-toolbar__separator"></span>
-        <div class="fd-popover fd-toolbar__overflow">
+        <div class="fd-popover fd-toolbar__overflow-popover">
```

Comment:
Please keep the prev naming. It's an unnecessary Breaking change

Reply:
`fd-toolbar__overflow`

Reply:
This class `fd-toolbar__overflow-popover` not required and removed.